File: C/1038.c
#include<stdio.h>
#include<stdlib.h>
#include<math.h>

int main(){
    int cod,quantidade;

    scanf("%d %d",&cod,&quantidade);
    switch (cod)
    {
    case  1:
        printf("Total: R$ %.2lf\n",quantidade*4.00);
        break;
    
    case 2:
        printf("Total: R$ %.2lf\n",quantidade*4.50);
        break;
    
    case 3:
         printf("Total: R$ %.2lf\n",quantidade*5.00);
         break;
    case 4:
        printf("Total: R$ %.2lf\n",quantidade*2.00);
         break;
    case 5:
        printf("Total: R$ %.2lf\n",quantidade*1.50);
        break;
    }


    return 0;
}

File: PY/1038.py
# BEE 1038 - Lanche - Python
def get_total(cod, quantidade):
    produtos = [4, 4.5, 5, 2, 1.5]
    return produtos[cod-1]*quantidade


cod, quantidade = input().split()
cod = int(cod)
quantidade = int(quantidade)

total = get_total(cod, quantidade)
print(f'Total: R$ {total:.2f}')
